src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/ExplicitRestoreReason.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio.Telemetry
```

Comment:
I prefer using assembly name as the namespace:

```suggestion
namespace NuGet.VisualStudio.Common
```